.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: ['3.6', '3.7', '3.8', '3.9', '3.10']
```

Comment:
This change isn't in the scope of this PR, please submit the changes to build.yml/tox.in in a separate PR.